s3-eventbridge-ecs/README.md

```diff
@@ -0,0 +1,81 @@
+
+# Requirements
+
+- [Create an AWS account](https://aws.amazon.com) if you do not already have, create them and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Terramform installed](https://www.terraform.io/)
+
+# Deployment Instructions
+
+1. Configure AWS CLI to the account and region you want to deploy the application to
+
+2. Create a new directory then navigate to that directory in a terminal and clone the GitHub repository:
+    ~~~ code
+    git clone https://github.com/aws-samples/serverless-patterns
+    
+3. Change directory to the pattern directory:
+    ~~~ code
+    cd serverless-patterns/s3-eventbridge-ecs
+    
+4. Go to the [Docker directory](src/docker/README.md) and execute the docker command and bash script to do the following:
+    - Create the ECR Repository
+    - Create the task image and push it to the repository
+    - Note the ARNs of the image name for the task. The ARN and image name will be needed in terraform script
+
+5. Make the following changes in the [terraform script](pattern_s3_eb_ecs.tf):
+    - Put an unique S3 bucket name in line 2
+    - Put the image URI in the line 4
+    - Put a subnet block in line 5
+    - Put the AWS region where the script will deploy resources in line 6
+
+    ~~~~ code
+    #Sample Configuration
+    locals {
+        bucket_name     = "mybucketname"
+        event_bus_name  = "default"
+        container_image = "123456789000.dkr.ecr.us-east-2.amazonaws.com/myimagename:latest"
+        ecs_subnet_id   = "subnet-00000000000000000"
+        region          = "us-east-1"
+
+    }
+
+6. Execute the following terraform command to deploy the stack
+
+    ~~~~ code
+    terraform init
+    terraform plan
+    terraform apply -auto-approve
+
+7. The terraform will create a ECS cluster and deploy the image (created in step 4) in ECS. The script will use the default EventBridge event bus and create S3 notification rule to the default event bus.The rule will listen to the S3 upload events in the incoming folder. The script will provision all the necessary IAM roles and policies to execute the pattern
+
+# Testing
+
+1. Once the terraform script executed successfully, create a folder called 'incoming' inside the newly created S3 bucket and upload a sample csv file
+
+2. The S3 file upload will trigger an EventBridge Rule which will call the ECS task. The ECS task will execute and print the csv data in the cloudwatch log
+
+3. Check the CloudWatch log group to see the ECS task execution details. the logs can be found in /ecs/serverlessland-dump-env-vars
```

Comment:
this step is part of the previous step from the user perspective.
can you show a CLI Command to check logs from terminal